docker/README.md

```diff
@@ -1,180 +1,5 @@
-# Open AD Kit: Containerized Workloads for Autoware
+# Open AD Kit: containerized workloads for Autoware
 
-Open AD Kit offers two types of Docker image to let you get started with Autoware quickly: `devel` and `runtime`.
+[Open AD Kit](https://autoware.org/open-ad-kit/) offers containers for Autoware to simplify the development and deployment of Autoware and its dependencies. This directory contains the Dockerfiles and scripts to build and run the containers.
```

Comment:
spell check is complaining about "Dockerfiles"
Please add the following line to ignore it:
```
<!-- cspell: ignore Dockerfile -->
```

Reply:
or you can just remove the word "Dockerfiles". It still makes sense.

```
This directory contains the scripts to build and run the containers.
```